File: Practice/src/exercise/WeaponsOfKnights.py
def solution(number, limit, power):
    answer = 0
    arr = [1] * number
    for i in range(2, number + 1):
        for j in range(1, int(number / i) + 1):
            arr[i * j - 1] += 1
    return sum(power if limit < i else i for i in arr)

File: Practice/src/exercise/WeaponsOfKnights.js
function solution(number, limit, power) {
    const arr = Array(number).fill(1);
    for (let i = 2; i <= number; i++) {
        for (j = 1; j * i <= number; j++) {
            arr[i * j - 1]++;
        }
    }
    return arr.reduce((p, c) => (limit < c ? p + power : p + c));
}
